apps/hash-api/src/graphql/resolvers/integrations/linear/sync-workspaces-with-teams.ts

```diff
@@ -46,8 +45,8 @@ export const syncLinearIntegrationWithWorkspacesMutation: ResolverFn<
     },
   );
 
-  const userAccountId = extractOwnedByIdFromEntityId(
-    linearIntegration.entity.metadata.recordId.entityId,
+  const userAccountId = extractAccountId(
+    linearIntegration.entity.metadata.recordId.entityId as AccountEntityId,
   );
```

Comment:
This change should be reverted, the entity here is not a user, it's a Linear Integration entity. extractOwnedByIdFromEntityId is correct